rskj-core/src/main/java/co/rsk/core/bc/PendingState.java

```diff
@@ -91,9 +152,9 @@ public BigInteger getNonce(RskAddress addr) {
     }
 
     private void executeTransactions(Repository currentRepository, List<Transaction> pendingTransactions) {
-        pendingTransactions.stream()
-            .sorted(TRANSACTION_COMPARATOR)
-            .forEach(pendingTransaction -> executeTransaction(currentRepository, pendingTransaction));
+
```

Comment:
I removed the old way to sort tansactions, I suppose it is okay but I wanted to just point this change to you guys pay attention .